setup.py

```diff
@@ -25,10 +25,13 @@
 # override these and pass --static for a static build. See
 # doc/build.txt for more information. If you do not pass --static
 # changing this will have no effect.
-STATIC_INCLUDE_DIRS = []
-STATIC_LIBRARY_DIRS = []
+def static_env_list(name):
+    return list(filter(None, map(lambda x: x.strip(), os.environ.get(name, "").split(","))))
```

Comment:
This is almost incomprehensible at first and second glance. Could you please turn this into a list comprehension?